torch/_inductor/codegen/triton.py

```diff
@@ -1176,6 +1197,9 @@ def can_fuse(self, node1, node2):
             if not (numel1 == numel2 and rnumel1 == rnumel2):
                 return False
 
+            if node1.is_template():
+                return True
```

Comment:
Can you add a comment why a template means an early return?

Reply:
Sure